packages/rest/src/parser.ts

```diff
@@ -129,9 +223,11 @@ function buildOperationArguments(
   }
 
   debug('Validating request body - value %j', body);
-  validateRequestBody(body, operationSpec.requestBody, globalSchemas);
+  validateRequestBody(body, operationSpec.requestBody, globalSchemas, {
+    coerceTypes: body.coercionRequired,
```

Comment:
Why are we passing `coerceTypes` in options, when `validateRequestBody` can infer the value from `body.coercionRequired`?